runtime/src/speedy-j.cpp

```diff
@@ -0,0 +1,229 @@
+#include <zsr/speedy-j.hpp>
+
+namespace speedyj
+{
+
+static std::string jsonEncoded(const std::string& s)
+{
+    std::string res;
+    res.reserve(s.size());
+
+    for (auto c : s) {
```

Comment:
Small nitpick: `auto const&`? Or do you think this should definitely be caught by the optimizer?